### PR TITLE
Fix: Token not showing on proposals with underlying token

### PR DESCRIPTION
**Pull request by josemarinas**
## Description

Fix token not appearing on proposals with underliying token

Task ID: [OS-559](https://aragonassociation.atlassian.net/browse/OS-559)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them when possible.
- [ ] I have updated the `CHANGELOG.md` file in the root folder of the package after the `[UPCOMING]` title and before the latest version.
- [ ] I have tested my code on the test network.

[OS-559]: https://aragonassociation.atlassian.net/browse/OS-559?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/client/src/tokenVoting/internal/utils.ts [hunk 1]
```diff
@@ -232,6 +237,21 @@ function parseToken(
...
       type: TokenType.ERC721,
     };
+  } else if (subgraphToken.__typename === SubgraphContractType.ERC20_WRAPPER) {
+    token = {
+      address: subgraphToken.id,
+      symbol: subgraphToken.symbol,
+      name: subgraphToken.name,
+      decimals: subgraphToken.decimals,
+      type: TokenType.ERC20,
+      underlyingToken: {
```
Comment (brickpop) [inline modules/client/src/tokenVoting/internal/utils.ts:247]: Don't we have a way for consumers to check the enum to determine that this is a wrapper? If checking whether `underlyingToken` exists is fine, then resolve the comment.

File: modules/client/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
```
Comment (brickpop) [inline modules/client/package.json:1]: Could we add some test to check `parseToken`?

File: modules/client/src/tokenVoting/internal/utils.ts (hunk 1 above)
Comment (josemarinas) [reply modules/client/src/tokenVoting/internal/utils.ts:247]: Yep, thats basically the way to do it, if underliyingToken exists is wrapped, else it isnt

File: modules/client/package.json (hunk 2 above)
Comment (josemarinas) [reply modules/client/package.json:1]: done ✨

**Review by brickpop — DISMISSED** [review]

**Review by brickpop — APPROVED** [review]

**Pull request merged**